src/Phinx/Migration/AbstractMigration.php

```diff
@@ -82,7 +82,7 @@
     /**
      * Class Constructor.
      *
-     * @param int $version Migration Version
+     * @param float $version Migration Version
```

Comment:
not int?

Reply:
It is declared as float in the phpdoc for the class property. Maybe because of size limitation for int in windows 32-bit platforms.